src/utils/configs.js

```diff
@@ -74,7 +75,8 @@ if (isLocalDevelopment) {
   localDevImages = {
     logo: appLogo,
     company_logo: companyLogo,
-    editor_logo: sceneEditorLogo
+    editor_logo: sceneEditorLogo,
+    home_background: homeHeroBackground
```

Comment:
nit: these local dev images were meant to be unbranded placeholders, so if we do need to use home-hero-background here, I would replace it with a generic image instead of one that says "mozilla" in it. Not a huge deal, but I've generally been trying to remove all branded assets from the client so that we don't worry about having to enforce users accidentally showing them on forks going forward.

Reply:
Replaced